Languages/French/Keyed/Manager_Keyed.xml

```diff
@@ -21,6 +22,26 @@
   <FM.OrderDown>Diminuer la priorité de travail</FM.OrderDown>
   <FM.OrderTop>Priorité maximale</FM.OrderTop>
   <FM.OrderBottom>Priorité minimale</FM.OrderBottom>
+  <multiple>[multiple]</multiple>
+  <FM.Threshold>Seuil</FM.Threshold>
+  <FM.CheckReachability>Vérifie si la cible est atteignable</FM.CheckReachability>
+  <FM.CheckReachability.Tip>Calcule si un trajet vers la cible esty possible (pour n'importe quel colon) avant de créer une désignation.</FM.CheckReachability.Tip>
+  <FM.Expensive.Tip>Activer cette option peu faire chuter les performances, particulierement sur les grandes cartes, les zones montagneuses, les tâches avec beaucoup de cibles potentielles et les colonies avec un grand nombre de colons.</FM.Expensive.Tip>
+  <FM.PathBasedDistance>Calcul la distance à partir du trajet actuel</FM.PathBasedDistance>
+  <FM.PathBasedDistance.Tip>Si activé, les distances seront calculées avec le vrai trajet. Si désactivé, les distances sont calculées à "vol d'oiseau". \n\nNOTE: Activer cette option sur des tâches avec un grand nombre de cibles possible peut générer des pics de ralentissement significatifs</FM.PathBasedDistance.Tip>
+  <FM.CountAllOnMap>Compter sur toute la carte</FM.CountAllOnMap>
+  <FM.CountAllOnMap.Tip>Si activé, compte tous les éléments autorisés par le filtre à seuil sur la carte. Si désactivé, ne compte que les stocks et les bâtiments de stockage.</FM.CountAllOnMap.Tip>
+  <FM.None>Aucun</FM.None>
+  <!-- 0: target (thing), 1: distance (int), 2: yield amount (int), 3: yield type (def) -->
+  <Fluffy.Manager.DesignationLabel>{0}\n  distance: {1}\n  production: {2}x {3}</Fluffy.Manager.DesignationLabel>
```

Comment:
are you sure this was translated? (My french is terrible, but this looks too english 😄 )